src/ValidationPackages/ValidationPackagesConfig.fs

```diff
@@ -22,15 +22,36 @@ type ValidationPackagesConfig(validation_packages, ?arc_specification) =
     member this.Copy() =
         ValidationPackagesConfig.make this.ValidationPackages this.ARCSpecification
 
-    override this.Equals(obj) =
-        match obj with
-        | :? ValidationPackagesConfig as other_vpc -> other_vpc.ValidationPackages = this.ValidationPackages && other_vpc.ARCSpecification = this.ARCSpecification
+
+    /// Pretty printer 
+    override this.ToString() =
+        [
+            if this.ARCSpecification.IsSome then $"arc_specification: {this.ARCSpecification.Value}"
+            "validation_packages:"
+            this.ValidationPackages
+            |> Seq.map (fun vp -> vp.ToString())
+            |> String.concat System.Environment.NewLine
+        ]
+        |> String.concat System.Environment.NewLine
+
+    member this.StructurallyEquals (other: ValidationPackagesConfig) =
+        let sort = Array.ofSeq >> Array.sortBy (fun (vp: ValidationPackage) -> vp.Name, vp.Version)
```

Comment:
Better to use ResizeArray.Sort, as conversions can be performance issue on transpiled code.

Reply:
I re-used the pattern introduced by ARCTable here:

https://github.com/nfdi4plants/ARCtrl/blob/dc3822fd1fec14c6009425e618531adf8828119a/src/Core/Table/ArcTable.fs#L697-L702

 I am pretty sure that the person that implemented this encountered the same problem as me when trying to use the `Sort` method of the resize array, as it will produce runtime errors which may be due to the default comparison not working on our types?

Reply:
Could be, yes. Alright!